### ISSUE TITLE
orchestrator-client requires curl and jq 

### ISSUE THREAD

**Issue by marcosvm** [openark/orchestrator#862]
`orchestrator-client` requires `curl` and `jq` to be run from inside the container, for example:

```
docker run -ti orchestrator:latest /usr/local/orchestrator/resources/bin/orchestrator-client
```

will give an error like:
```
orchestrator-client[1]: cannot find curl
```

**Comment by zoidyzoidzoid** [comment]
Solved by #863?

**Comment by marcosvm** [comment]
Yes, we can close this.

**Comment by shlomi-noach** [comment]
Pro tip: for next PRs, instead of writing 

> Related issue: #862

Use the magic word

> Fixes #862

This will auto-close the issue when the PR is merged.
